.gitignore

```diff
@@ -1,2 +1,7 @@
 softhsm2.conf
 softhsm/
+*.txt
+*.pem
```

Comment:
Probably better for these to be ceremonies/**.pem or something like that? I don't exactly recall the gitignore glob syntax.